gerousiabot/discord_apis.py

```diff
@@ -2,7 +2,57 @@
 This module will hold the functions used for
 interacting with the Discord APIs
 """
+import hikari
+import asyncio
+import os
+from dotenv import load_dotenv
 
+def get_api_token(key) -> str:
+    """
+    Gets the API token for a given key
+    :param key: the key for the token you want returned
+    :return: the API token
+    """
+    load_dotenv()
+    return os.getenv(key)
+
+def get_rest_client():
+    rest_app = hikari.RESTApp()
+    token = get_api_token("DISCORD_API_KEY")
+    rest_client = rest_app.acquire(token, "Bot")
+    return rest_client
+    
+
+async def get_my_user(rest_client):
+    async with rest_client as client:
```

Comment:
Can we avoid the aliasing by importing `client` from the `hikari` package, then including the type in the parameter?
`def get_my_user(rest_client: client) `